tests/foreman/cli/test_import.py

```diff
@@ -0,0 +1,110 @@
+from robottelo.cli.org import Org
+
+
+@ddt
+def csv_to_dataset(csv):
+    """Converts the csv with header as a first entry to a python dict"""
+    keys = csv[0].split(',')
+    del csv[0]
+    dataset = list()
+    for val in csv:
+        dataset.append(
+            dict(zip(keys, val.split(',')))
+        )
+    return dataset
+
+
+class TestImport(CLITestCase):
+    """Import CLI tests.
+
+    All default tests pass no options to the imprt object
+    In such case methods download a default data set from URL
+    specified in robottelo.properties.
+
+    """
+    def test_import_orgs_default(self):
+        """@test: Import all organizations from the default data set
+        (predefined source).
+
+        @feature: Import Organizations
+
+        @assert: 3 Organizations are created
+
+        """
+        files = prepare_import_data()[1]
+        rc1 = Import.organization(
+            {
+                'csv-file': files['users']
+            }
+        )
+        rc2 = ssh.command('cat {0}'.format(files['users']))
+        # pop the array as ssh.command appends 1 empty item at the end
+        rc2.stdout.pop()
+
+        # now to check whether the orgs from csv appeared in sattelite
+        orgs = set(o['name'] for o in Org.list().stdout)
+        imporgs = set(o['organization'] for o in csv_to_dataset(rc2.stdout))
+
+        self.assertEqual(rc1.return_code, 0)
+        self.assertEqual(
+            rc1.stdout,
+            [
+                u'Summary',
+                u'  Created {0} organizations.'.format(len(imporgs)),
+                u''
+            ]
+        )
+        self.assertEqual(False in [o in orgs for o in imporgs], False)
+
+    def test_import_users_default(self):
+        """@test: Import all 3 users from the our default data set (predefined
+        source).
+
+        @feature: Import Users
+
+        @assert: 3 Users created
+
+        """
+        tmpdir, files = prepare_import_data()
+        pwdfile = tmpdir + '/' + gen_string('alpha', 6)
+
+        rc = Import.user(
+            {
+                'csv-file': files['users'],
+                'new-passwords': pwdfile
+            }
+        )
+        self.assertEqual(rc.return_code, 0)
+        self.assertEqual(rc.stdout, [u'Summary', u'  Created 3 users.', u''])
+
+    def test_reimport_orgs_default(self):
+        """@test: Try to Import all organizations from the
+        predefined source and try to import them again
+
+        @feature: Import Organizations twice
+
+        @assert: 2nd Import will result in No Action Taken
+
+        """
+        files = prepare_import_data()[1]
+        self.assertEqual(Import.organization(
+            {
+                'csv-file': files['users']
+            }
+        ).return_code, 0)
+        self.assertEqual(
+            Import.organization(
+                {
+                    'csv-file': files['users']
+                }
+            ).stdout,
+            [u'Summary', u'  No action taken.', u'']
```

Comment:
I can say that if we change languages most of CLI tests will fail. When the language is change all the expected field changes and will fail, for example, `content-host['name']` in Portuguese will be `content-host['nome']` this will raise an `KeyError` exception. With that said checking the output is a plus and we will need to rework the entire framework to support other langues. I think we can leave this assertion for now.